modules/lcov/2.2/MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+module(name = "lcov", version = "2.2")
```

Comment:
please add `bazel_compatibility = [">=7.2.1"]` for overlay usage

Reply:
added